### ISSUE TITLE
Broken pycbc_single_template followups

### ISSUE THREAD

**Issue by titodalcanton** [ligo-cbc/pycbc#726]
Hi Ian,

a workflow I ran with recent master had all single_template_plot jobs failing with this error:

```
Traceback (most recent call last):
  File "/home/tito/sw_install/pycbc_latest//bin/pycbc_single_template_plot", line 4, in <module>
    __import__('pkg_resources').run_script('PyCBC===c33e46', 'pycbc_single_template_plot')
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 735, in run_script
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 1652, in run_script
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-c33e46-py2.7.egg/EGG-INFO/scripts/pycbc_single_template_plot", line 80, in <module>
    newsnr = pycbc.events.newsnr(snr, chisq)
  File "/home/tito/sw_install/pycbc_latest/lib/python2.7/site-packages/PyCBC-c33e46-py2.7.egg/pycbc/events/events.py", line 182, in newsnr
    return newsnr[0]
IndexError: index 0 is out of bounds for axis 0 with size 0
```

Test case on Atlas at ~tito/o1/imbh/runs/phenomd_bank_1/single_template_debug.

From an initial check it looks like there is an inconsistency in pycbc_single_template between the trig_start_time that's returned by select_segments() and what is calculated in [line 240](https://github.com/ligo-cbc/pycbc/blob/master/bin/pycbc_single_template#L240), which causes the wrong slice of SNR to be saved to the HDF file. pycbc_single_template_plot then fails because the SNR time series is empty. I'm not sure whether the bug is in pycbc_single_template or somewhere above in the workflow though.

**Comment by titodalcanton** [comment]
Note that if I run pycbc_single_template with manual input of --gps-{start,end}-time, I don't get the error.

**Comment by spxiwh** [comment]
Hi @titodalcanton thanks, I've identified the bug and will post a pull request soon. The issue was at line 240 as you identified.

As an aside, the reason you hit this and the test workflow did not was because you are not using the `--window` option to single_template (changing this from an edge-case bug to an always-see-it bug). Not using this option will result in a *lot* of unnecessary data being written and might make you unpopular on atlas. Is there a reason *not* to use this option? It in no way affects the filtering, it just stores less data.

**Comment by spxiwh** [comment]
This should be fixed by #727.
